internal/config/config.go

```diff
@@ -63,9 +63,10 @@ type API struct {
 }
 
 type EKS struct {
-	AccountID   string `mapstructure:"account_id"`
-	Region      string `mapstructure:"region"`
-	ClusterName string `mapstructure:"cluster_name"`
+	AccountID                    string `mapstructure:"account_id"`
+	Region                       string `mapstructure:"region"`
+	ClusterName                  string `mapstructure:"cluster_name"`
+	AWSClientTimeoutMilliseconds int    `mapstructure:"aws_client_timeout_milliseconds"`
```

Comment:
no need for `AWS` prefix in `EKS` section; change type to time.Duration, maybe `APITimeout time.Duration`.

See `Controller.InitialSleepDuration` for how constants are set via api for this.